hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/contract/hapi/ContractCallSuite.java

```diff
@@ -364,7 +365,7 @@ final HapiSpec lowLevelEcrecCallBehavior() {
         final var TEST_CONTRACT = "TestContract";
         final var somebody = "somebody";
         final var account = "0.0.1";
-        return defaultHapiSpec("LowLevelEcrecCallBehavior", NONDETERMINISTIC_TRANSACTION_FEES)
+        return defaultHapiSpec("LowLevelEcrecCallBehavior", FULLY_NONDETERMINISTIC)
```

Comment:
Just for my information, what additional laxity of `FULLY_NONDETERMINISTIC` (over `NONDETERMINISTIC_TRANSACTION_FEES`) is needed by this test?  And also the next test?

Reply:
@david-bakin-sl 
Those inconsistencies in regards to the issues we are fixing are found in mono. And since the fuzzy matching is making a comparison between mono and modu results, they won't match because of the change in the behaviour. And as I'm aware the fuzzy match test itself was already successful and it is not a present thing that we are trying to be compatible with, if that makes sense..